recipes-support/console-setup/console-setup-ckbcomp-native_1.192.bb

```diff
@@ -1,15 +1,14 @@
-SRC_URI[md5sum] = "aa522a2fc8f596fd71f51e581b7471dc"
-SRC_URI[sha256sum] = "4dbe2c9559ff68a61315efe4de2cb932396ede8c4a1df4b46b6fb0846cc59d5b"
 DESCRIPTION = "Convert XKB keymap to console keymap"
 LICENSE = "GPLv2"
 LIC_FILES_CHKSUM="file://COPYRIGHT;md5=49cab1cfd397b014807c5b2bcc63e04f"
 RDEPENDS_${PN} = "perl-native"
 
 # Don't bother building the whole package. We only want the ckbcomp script.
 
-SRC_URI = "${DEBIAN_MIRROR}/main/c/console-setup/console-setup_${PV}.tar.xz"
+SRC_URI = "git://salsa.debian.org/installer-team/console-setup.git;protocol=https;tag=${PV}"
+SRCREV = "${PV}"
```

Comment:
You may want to specify the commit hash:
> Note that if you want to build a fixed revision and you want to avoid performing a query on the remote repository every time BitBake parses your recipe, you should specify a SRCREV that is a full revision identifier and not just a tag.

from: https://www.yoctoproject.org/docs/latest/ref-manual/ref-manual.html#var-SRCREV